composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.4",
         "league/flysystem": "~1.0",
-        "egeloen/http-adapter": "~0.7"
+        "php-http/httplug": "~1.1"
```

Comment:
It would probably be a good idea to add the virtual package as a dependency. http://docs.php-http.org/en/latest/httplug/library-developers.html#manage-dependencies